### PR TITLE
Integrate UNet predictions with batchlib

### PR DESCRIPTION
**Pull request by wolny**
- [x] add normalization function used by UNet2D
- [x] make it work with `TorchPrediction`

I've also changed `batchlib/segmentation/torch.py` to `batchlib/segmentation/torch_prediction.py` due to name clashing while importing `torch` in other files :)

### CODE REVIEW

File: batchlib/segmentation/torch_prediction.py [hunk 1]
```diff
@@ -77,6 +91,9 @@ def run(self, input_files, output_files, gpu_id=None, batch_size=1, normalize=de
             model = self.load_model(device)
             model.to(device)
 
-        input_batches, output_batches = files_to_jobs()
-        for in_batch, out_batch in tqdm(zip(input_batches, output_batches), total=len(input_files)):
-            self.predict_images(in_batch, out_batch, model, normalize, device)
+            #input_batches, output_batches = files_to_jobs()
```
Comment (wolny) [inline batchlib/segmentation/torch_prediction.py:94]: here I don't quite got the semantics of `files_to_jobs` so just commented it out for mow

Comment (constantinpape) [reply batchlib/segmentation/torch_prediction.py:90]: Ah I guess I forgot to fill that in :D. I think what I wanted to do here does not quite work.
I will just merge and fix it myself, that's much easier.

**Comment by wolny** [comment]
> Thanks a lot!

**Comment by constantinpape** [comment]
Ok, looks all good now, thanks!

**Pull request merged**